marimo/_plugins/ui/_impl/table.py

```diff
@@ -68,6 +68,7 @@ class ColumnSummary:
 @dataclass
 class ColumnSummaries:
     summaries: List[ColumnSummary]
+    is_disabled: Optional[bool] = None
```

Comment:
Doesn't `summaries` being the empty list imply that column summaries are disabled? In which case we could remove the `is_disabled` parameter.

Reply:
it is helpful to know if there are no summaries vs it actually being disabled. 

it could have no summaries if they failed for some reason or summaries are not supported in the table type

Reply:
Thanks @mscolnick for the explanation!

A better way is to implement a generic error handling for Function objects, so we can pass structural error information from the backend. Actually I even wanted to add the upper limit number to the `ColumnSummaries` object so there will be no hardcoded text in frontend (I didn't because it's even uglier than adding `is_disabled`).

Reply:
I agree structured errors would be great. we don't have great e2e type-safety yet for structured errors, so its fine to put the details inside the 200 response. 

i think you can add the upper limit as well. that seems like a good idea in case its configurable later.

Reply:
keeping in sync via comments is also totally great, this looks great!